### PR TITLE
Bump llvm-project to 820e65f9e2369d2990fde4b3e7cfceb64f0df9c8

### PR DESCRIPTION
**Pull request by silvasean**
Date:   Mon Oct 12 11:26:50 2020 -0700

### CODE REVIEW

**Review by stellaraccident — APPROVED** [review]
Personally, I don't feel like these trivial version bumps need review/approval. Maybe just a post-submit FYI and save review for the non-trivial cases?

**Comment by silvasean** [comment]
Agreed on the review. I was mostly just waiting for the CI checks before submitting.

**Pull request merged**
